hibike/devices/YogiBear/ui.h

```diff
@@ -0,0 +1,31 @@
+#ifndef UI_H
+#define UI_H
+
+// function prototypes
+void manual_ui();
+
+//A space for constants.
+extern float pwmInput; //Value that is received from hibike and is the goal PWM
+extern float pwmPID; //Value that is generated by PID, if PID is disabled, this will be equal to pwmInput
+extern float pwmOutput; //Value controlled by current limiting and is what is written to the motor
+extern float current_threshold; //threshold in amps
+extern volatile signed long old_encoder0Pos;
+extern double pos; //this used to be volatile.  Changed to not be float, so it'd compile.
+extern double vel;
```

Comment:
`pos` and `vel` are declared extern in all of the headers, but they're never actually defined anywhere (that I see). How does this even work?